tests/test_cff2pages.py

```diff
@@ -62,13 +63,13 @@ def test_authors_affiliation(self):
 expected_minimal_body = """<body>
 <div class="container">
     <div class="content">
-        
+
         <h1 class="blog-title"> Test CFF</h1>
 
-        
+
     <h2>
             Muster Mina
-            <sup>2</sup>, 
```

Comment:
Are the test failures related to my text editor stripping out extraneous whitespace, @B3J4y?